packages/sit-onyx/src/components/OnyxRipple/OnyxRipple.vue

```diff
@@ -0,0 +1,90 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { useRipple, type RippleConfig } from "../../composables/useRipple";
+import { type OnyxRippleProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxRippleProps>(), {
+  enterDuration: "300ms",
+  leaveDuration: "100ms",
+  color: "var(--onyx-ripple-color, var(--onyx-color-base-primary-600))",
+  terminateOnPointerUp: false,
+});
+
+const config = computed<RippleConfig>(() => ({
+  color: props.color,
+  container: rippleTrigger,
+  terminateOnPointerUp: props.terminateOnPointerUp,
+}));
+
+const rippleTrigger = ref<HTMLElement>();
+
+const { ripples, hideRipple, events } = useRipple(config);
```

Comment:
In combination with the other comments in this PR, would it be possible to make the animation CSS only?

Reply:
the animation is CSS only, only the creation of the ripples and some small calculations are Javascript.